tools/test-api

```diff
@@ -121,5 +122,17 @@ with test_server_running(
     )
     test_user_account_deactivated(client)
 
+    # Test realm deactivated error
+    guest_user.realm.deactivated = True
+    guest_user.is_active = False
+    guest_user.save()
```

Comment:
Same `do_deactivate_user` comment here -- we try to avoid manually changing objects like this because it can fail to apply other important side effects.

Reply:
Made these changes.